public/admin/config.yml

```diff
@@ -644,3 +644,24 @@ collections:
           - label: Metadata description
             name: metadataDescription
             widget: text
+  - label: Vaccinations directory
+    name: vaccines
+    folder: 'public/cms-content/vaccines'
+    create: true
+    label_singular: Vaccine
+    description: description
+    format: json
+    identifier_field: state
+    fields:
+      - label: State
+        name: state
+        widget: select
+        options:
+          - { label: "Massachussetts", value: "MA" }
+      - label: More Info URL
+        name: moreInfoUrl
+        widget: string
+      - label: Sign Up URL
+        name: signUpUrl
+        widget: string
+        
```

Comment:
Also, it might be simpler to keep the vaccination info for all the states together in one file since we will be loading them dynamically anyways. One of the possible designs for vaccination data is to have a dedicated page for vaccinations that will need info from all the states at once